Storage/SQL/SQLiteRemoteDevices.swift

```diff
@@ -0,0 +1,69 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import Shared
+import XCGLogger
+import Deferred
+import Account
+
+private let log = Logger.syncLogger
+
+open class SQLiteRemoteDevices: RemoteDevices {
```

Comment:
This is just a way of hanging a single method on a class, but you're making an instance out of it. Why not make this `extension SQLiteRemoteClientsAndTabs: RemoteDevices`? After all, you've now tightly coupled the former with the latter, no?